pkg/util/flag/flags.go

```diff
@@ -109,7 +110,7 @@ func (v IPPortVar) Set(s string) error {
 	if net.ParseIP(host) == nil {
 		return fmt.Errorf("%q is not a valid IP address", host)
 	}
-	if _, err := strconv.Atoi(port); err != nil {
+	if _, err := netutils.ParsePort(port, false); err != nil {
```

Comment:
I think here we want to pass true.  We don't know if the consumer of this flag intends :0 to be allowed or not, so they will have to re-parse it when they consume it.